### ISSUE TITLE
expired markets shouldn't be linked as related + unrelated market is being incorrectly linked

### ISSUE THREAD

**Issue by matt-bullock** [AugurProject/augur#9401]
See below:

Why is this market related

![image](https://user-images.githubusercontent.com/39496225/95323753-f39deb80-089e-11eb-8408-7b612cd47081.png)

**Comment by bthaile** [comment]
this is working correctly. The markets are grouped by template market title inputs which is `2020`. So these are 2020 political markets therefore they are grouped. 

recommend closing

**Comment by matt-bullock** [comment]
should expired markets be linked to open markets though?

**Comment by JohnDanz** [comment]
yes, according to the mocks there are situations where an expired market would be grouped with open markets. like a combo market card where the money line has a tentative winner but spread/overunder might not have reported yet.

**Comment by Chwy5** [comment]
We should not show any related market that is resolved (expired), for any futures markets
